README.md

```diff
@@ -1,15 +1,126 @@
-# Holochain Command Line
+# Holochain Command Line Tools
 
-Holochain is being rewritten in Rust as a set of crates that make up the Holochain Library. This means we need a container app to load Holochain and use its features. Using the cmd line wrapper [CLAP](https://github.com/kbknapp/clap-rs ) it was easy to parse the command line and pass those values to Holochain.
-As the library progresses we will add more exposed functions to the command line.
+`holochain-cmd` is a set of tools for building and running [Holochain](https://holochain.org) applications from the command line. The tools are written in Rust, and delivered as binary executables.
+
+## How To Install
+
+Prerequisite: [Rust](https://www.rust-lang.org/en-US/install.html) must be installed on your computer
+
+To install the Holochain command line , run the following commands in a terminal
+```shell
+$ git clone https://github.com/holochain/holochain-cmd.git
+$ cd holochain-cmd
+$ cargo install -f --path .
+```
+
+The command line tools are now available in your command line using the `hcdev` command.
+Run `hcdev -V` to confirm.
+Run `hcdev help` for help.
+
+## Available Commands
+
+`(u)` means the command is as-yet unimplemented.
+
+| Command   | Use                                                                 |
+|-----------|---------------------------------------------------------------------|
+| agent (u) | Starts a Holochain node as an agent                                 |
+| generate  | Generates a new zome and scaffolds the given capabilities           |
+| init      | Initializes a new Holochain app at the given directory              |
+| package   | Builds the current Holochain app into a `.hcpkg` file               |
```

Comment:
The output is currently called `bundle.json`

Reply:
Yeah... you can also customize the name of the output with a `-o` flag though... 

Anyways, I just borrowed that text from what Julian had in the `hcdev help` command